tests/test_project.py

```diff
@@ -1480,85 +1480,71 @@ def test_import_own_project(self):
             assert len(tmp_project) == len(self.project)
 
 
-class TestProjectRepresentation(TestProjectBase):
+def add_jobs_homogeneous(project, num_jobs):
+    # Add jobs with many different state points
+    valid_sp_values = [None, 0, 1, 0.0, 1.0, True, False, [0, 1, 2], [0, 1.0, False]]
```

Comment:
If the goal is to always test the same values for `valid_sp_values` for `add_jobs_homogeneous` and `add_jobs_heterogeneous`, would it make sense to define `valid_sp_values` outside of both functions and take them in as inputs?

Reply:
Good idea! I'll define a module-level constant.